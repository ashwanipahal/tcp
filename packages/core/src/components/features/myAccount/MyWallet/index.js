export { default } from './container/MyWallet.container';
