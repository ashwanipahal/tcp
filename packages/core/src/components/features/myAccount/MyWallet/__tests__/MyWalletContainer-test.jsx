// Boiler plate for furthur test cases to be written

import { shallow } from 'enzyme';
import React from 'react';
import MyWallet from '../container/MyWallet.container';

describe('MyWallet Container', () => {
  it('should render MyWallet Correctly', () => {
    const mainContent = jest.fn();
    const tree = shallow(<MyWallet mainContent={mainContent} />);
    expect(tree).toMatchSnapshot();
  });
});
