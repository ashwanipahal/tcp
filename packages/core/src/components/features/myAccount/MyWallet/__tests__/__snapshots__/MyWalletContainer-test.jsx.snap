// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWallet Container should render MyWallet Correctly 1`] = `ShallowWrapper {}`;
